tests/test_db_utils_close_manager_coverage.py

```
@@ -0,0 +1,118 @@
+                "mmrelay.db_utils.get_db_path",
+                return_value="/nonexistent/path/db.sqlite",
+            ),
+            patch(
+                "mmrelay.db_utils._resolve_database_options",
+                return_value=(True, 5000, {}),
+            ),
+        ):
+            with patch(
+                "mmrelay.db_utils.DatabaseManager",
+                side_effect=RuntimeError("init failed"),
+            ):
```

Ruff (0.15.10)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)